packages/file_selector/file_selector_macos/example/lib/get_directory_page.dart

```diff
@@ -0,0 +1,76 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:file_selector_platform_interface/file_selector_platform_interface.dart';
+import 'package:flutter/material.dart';
+
+/// Screen that shows an example of getDirectoryPath
```

Comment:
See comments on the [Windows patch](https://github.com/flutter/plugins/pull/4424#pullrequestreview-862798634)